torchvision/transforms/functional.py

```diff
@@ -123,7 +123,7 @@ def _is_numpy_image(img: Any) -> bool:
     return img.ndim in {2, 3}
 
 
-def to_tensor(pic) -> Tensor:
+def to_tensor(pic: Image | np.ndarray) -> Tensor:
```

Comment:
Thanks for the PR @adamtheturtle , I think you'll have to use `Union` instead of `|` as `|` isn't supported in some older Python versions

Reply:
Thank you @NicolasHug . I have updated this PR.